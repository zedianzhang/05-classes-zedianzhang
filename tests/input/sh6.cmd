:load tests/input/tFold.hs
foldr add 0 myList
:quit