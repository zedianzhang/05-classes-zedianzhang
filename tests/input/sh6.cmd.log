nano: TBD: main
CallStack (from HasCallStack):
  error, called at src/Main.hs:10:3 in main:Main
