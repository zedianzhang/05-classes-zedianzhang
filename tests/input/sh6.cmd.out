------------------------------------------------------------
-------- The NANO Interpreter v.0.0.0.0 --------------------
------------------------------------------------------------

λ [0] definitions: tail head foldr add myList
λ [1] 15
λ [2] Goodbye.
